pkgs/development/tools/build-managers/bob/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, buildGoModule, fetchFromGitHub, git }:
+
+buildGoModule rec {
+  pname = "bob";
+  version = "0.5.3";
+
+  src = fetchFromGitHub {
+    owner = "benchkram";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "sha256-JG1fysCqqd/MwpNhKJwLr4cTGq4/88f9OMMapb+r3bc=";
+  };
+
+  vendorSha256 = "sha256-R+zXGR5isoo76oc4lsFf9uCM0Kyi8dQiKEg4BUxtv+k=";
+
+  excludedPackages = ["example/server-db" "test/e2e" "tui-example"];
+
+  # tests require network access
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A build system for microservices";
+    homepage    = "https://bob.build";
+    license     = licenses.asl20;
+    platforms   = platforms.unix;
+    maintainers = [ "zuzuleinen" ];
```

```suggestion
    maintainers = with maintainers; [ zuzuleinen ];
```
And add yourself to maintainer-list.

---

Sure, https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix

---

Here are some common starting points with contribution guidelines.

https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers

https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes